frontend/components/FieldAsset/FieldAsset.css

```diff
@@ -0,0 +1,28 @@
+.container {
+  width: 100%;
+  margin-top: 9px;
+  height: 40px;
+  line-height: 25px;
+  font-size: 15px;
+  box-sizing: border-box;
+  border: 1px solid #e4e4e4;
+  border-radius: 5px;
+  outline: 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.container[preview] {
```

Comment:
Yes, I think we need to be a bit more consistent here. Do that. Add to style guide too if you've the time.